src/Components/WebAssembly/WebAssembly/src/Services/WebAssemblyConsoleLogger.cs

```diff
@@ -98,12 +98,7 @@ private void WriteMessage(LogLevel logLevel, string logName, int eventId, string
                             _jsRuntime.InvokeVoid("console.error", formattedMessage);
                             break;
                         case LogLevel.Critical:
-                            // Writing to Console.Error is even more severe than calling console.error,
-                            // because it also causes the error UI (gold bar) to appear. We use Console.Error
-                            // as the signal for triggering that because it's what the underlying dotnet.wasm
-                            // runtime will do if it encounters a truly severe error outside the Blazor
-                            // code paths.
-                            Console.Error.WriteLine(formattedMessage);
+                            _jsRuntime.InvokeVoid("Blazor._internal.dotNetCriticalError", formattedMessage);
```

Comment:
Unmarshalled?

Reply:
OK, that's a good idea. Most times it will make no difference, but I guess if the exception is something like "S.T.J. is somehow completely broken" then we don't want to rely on S.T.J. during the reporting of that error.

Done.